.github/workflows/bump-datadog-ci.yml

```
@@ -10,26 +14,32 @@ on:
         type: string
         default: 'latest'
 
-permissions:
-  contents: write
-  pull-requests: write
-
 jobs:
   bump-datadog-ci:
     runs-on: ubuntu-latest
     steps:
       # Do the changes
+      - name: Get GitHub App token
+        id: get_token
```

The other ids seem to be in kebab case.
```suggestion
        id: get-token
```